rules/integrations/aws/privilege_escalation_sts_assumerole_usage.toml

```diff
@@ -29,7 +29,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:sts.amazonaws.com and event.action:AssumedRole and
+event.dataset:aws.cloudtrail and event.provider:sts.amazonaws.com and event.action:AssumeRole and
```

Comment:
```suggestion
event.dataset:aws.cloudtrail and event.provider:sts.amazonaws.com and event.action: (AssumeRole or AssumedRole) and
```

We have some global telemetry that shows events where `event.action` is indeed `AssumedRole`. Without a deeper dive, I assume the difference depends on pre vs post authorized/authenticated.

Since this workflow is legitimate and  used to retrieve temporary credentials, changing it to `AssumeRole` which occurs during the request, may indeed increase alert volume. Therefore, we need to simulate these in a test environment to understand what additional logic can be added so this rule is more threat-centric.

<img width="676" alt="Screenshot 2023-07-06 at 10 25 33 AM" src="https://github.com/elastic/detection-rules/assets/99630311/38e637dd-dd34-4774-a66d-d6b411ad5bb0">

Reply:
I have confirmed that `AssumeRole` is correct here related to the AWS STS service for this event.action.